### PR TITLE
docs: enable chinese link

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Enables the Chinese link in the translation menu. The files are not yet 100% translated, but the two that focus on our translation flow *are*. Having these available could potentially increase our translation contributions.

### CODE REVIEW

**Comment by raisedadead** [comment]
I think we should add Portuguese which has hit 100% (translated, not-proofread) as well.

**Comment by nhcarrigan** [comment]
Sure - I'll need to modify the workflow to allow those files to be downloaded, migrate the English files into a Portuguese folder (to avoid 404s until translations are approved) and add the link.

Given the scope, should I do that in a separate PR so as to avoid blocking this one?

**Comment by raisedadead** [comment]
Sounds good.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
